UsaEPay.NET/Factories/UsaEPayRequestFactory.cs

```diff
@@ -451,5 +451,19 @@ public static UsaEPayGetRequest RetrieveTokenDetailsRequest(string tokenId)
                 Endpoint = $"tokens/{tokenId}"
             };
         }
+        public static UsaEPayGetRequest RetrieveBatchListRequest()
+        {
+            return new UsaEPayGetRequest
+            {
+                Endpoint = $"batches"
+            };
+        }
+        public static UsaEPayGetRequest RetrieveBatchListByDateRequest(long openedge, long openedlt)
```

Comment:
Let's improve on these variable names, make them more easy to understand. We also need our comments on all this.